recipes/cuda-compiler/meta.yaml

```
@@ -0,0 +1,43 @@
+package:
+  name: cuda-compiler
+  version: 12.0.0
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_documentation/LICENSE.txt
+  sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
+
+build:
+  number: 0
+  skip: true  # [osx or win]
```

Think we need this `skip` to fix CI

```suggestion
  skip: true  # [osx or win]
  skip: true  # [cuda_compiler_version != "None"]
```